src/components/shop/scrollable-list/scrollable-list-covers.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @module core/layout/scrollable-list-covers
+ * @copyright BBC 2021
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import { getPanelY } from "./scrollable-panel.js";
+
+const getCoverPositions = (scene, covers, padding) => {
+    const safeArea = getPanelY(scene);
+    return {
+        top: covers.top
+            ? {
+                  y: safeArea.y - safeArea.height / 2 + padding.y,
+              }
+            : null,
+        bottom: covers.bottom
+            ? {
+                  y: safeArea.y + safeArea.height / 2 - padding.y,
+              }
+            : null,
+    };
+};
+
+const getCoverPadding = config => {
+    return { y: config.y * config.outerPadFactor, x: config.x * config.outerPadFactor };
+};
+
+export const resizeCovers = (scene, panel, covers, config) => {
+    if (panel && covers) {
+        const padding = getCoverPadding(config);
+        const positions = getCoverPositions(scene, covers, padding);
+        const panelWidth = panel.getChildrenWidth();
+        covers.top &&
+            covers.top
+                .setX(0 - padding.x)
+                .setY(positions.top.y)
+                .setScale(panelWidth / covers.top.width);
+        covers.bottom &&
+            covers.bottom
+                .setX(0 - padding.x)
+                .setY(positions.bottom.y)
+                .setScale(panelWidth / covers.bottom.width);
+    }
+};
+
+export const createCovers = (scene, config) => {
+    if (scene.config.listCovers) {
+        const top = config.top ? scene.add.image(0, 0, config.top.key) : null;
+        const bottom = config.bottom ? scene.add.image(0, 0, config.bottom.key) : null;
+        return { top, bottom };
+    }
+    return false;
+};
```

Comment:
probably can make this a one liner, something like this?
```export const createCovers = (scene, config) => scene.config.listCovers && { top: config.top && scene.add.image(0, 0, config.top.key), bottom: config.bottom && scene.add.image(0, 0, config.bottom.key) }```

having scene.config and config bit confusing to read

Reply:
I was considering renaming `config` to `coversConfig`

Reply:
I originally wrote it as returning the object but it didn't seem as clear.

Reply:
I was making a compromise but then ended up writing the same one-liner. I've put it over multiple lines as it's clearer